src/wizard/obtain_source.jl

```diff
@@ -422,14 +421,14 @@ function step2(state::WizardState)
     if yn_prompt(state, "Do you want to customize the set of compilers?", :n) == :y
         get_compilers(state)
         # Default GCC version is the oldest one
-        get_preferred_version(state, "GCC", getversion.(available_gcc_builds))
+        get_preferred_version(state, "GCC", getversion.(BinaryBuilderBase.available_gcc_builds))
         # Default LLVM version is the latest one
-        get_preferred_version(state, "LLVM", getversion.(reverse(available_llvm_builds)))
+        get_preferred_version(state, "LLVM", getversion.(reverse(BinaryBuilderBase.available_llvm_builds)))
     else
         state.compilers = [:c]
         # Default GCC version is the oldest one
-        state.preferred_gcc_version = getversion(available_gcc_builds[1])
+        state.preferred_gcc_version = getversion(BinaryBuilderBase.available_gcc_builds[1])
         # Default LLVM version is the latest one
-        state.preferred_llvm_version = getversion(available_llvm_builds[end])
+        state.preferred_llvm_version = getversion(BinaryBuilderBase.available_llvm_builds[end])
```

Comment:
Can you just `using BinaryBuilderBase: available_gcc_builds, available_llvm_builds` so that you don't have to litter the module name all over the place?

Reply:
Yep, done :+1: